src/test/java/com/coherent/task30/YandexLogin.java

```diff
@@ -1,67 +1,44 @@
 package com.coherent.task30;
 
+import com.coherent.task40.BaseTest;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.openqa.selenium.By;
-import org.openqa.selenium.Dimension;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
-
 import java.time.Duration;
-import java.util.concurrent.TimeUnit;
-
 
-public class YandexLogIn {
+import static com.coherent.task40.TestConstants.YANDEX_LOGIN_PAGE;
 
-    private WebDriver driver;
+public class YandexLogin {
+    WebDriver driver = new ChromeDriver();
     Locators locator = new Locators();
-    String expectedUserName = "fine.lname";
+    private String expectedUserName = "fine.lname";
+    private String username = "fine.lname";
```

Comment:
Variables have the same value. Can be replaced by one variable

Reply:
done